queries/vue/injections.scm

```diff
@@ -2,7 +2,7 @@
   (style_element
     (start_tag) @_no_attribute
     (raw_text) @css)
-  (#match? @_no_attribute "<\\s*style\\s*>")
+  (#match? @_no_attribute "^\\<\\s*style\\s*\\>")
```

Comment:
Unsure why, but when the `<` character was not explicitly escaped, the whole match returns true no matter the content inside